integr8ly/installation
# Add the walkthrough webapp install role

To test this with a local `oc cluster up`,

```
ansible-playbook -K -i roles/webapp/tests/inventory roles/webapp/tests/test-install.yml
```

If testing in a different cluster, you will need to set the `openshift_master_config_path`
var back to `/etc/origin/master/master-config.yaml` or use a different inventory file.

After install, there should be a new namespace `webapp` with the tutorial-web-app deployed
and accessible at a route.

At this time the version of the webapp deployed will be using a mock list of services with incorrect urls.
This is expected and will change at a later point.
